content/news/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: News
+date: 2019-02-21
+publishdate: 2019-02-21
+---
+
+{{< newsroom/news
+      id="news-list-container"
+      publishTarget="openmdm"
```

Comment:
You mentioned that you are using the no-mh template, but it isn't referenced here. Did you maybe not commit that change?

Reply:
This file already exists in node_module/hugo-theme, I think it's auto referring to it, is not?

Reply:
Is that I have to pass  $templateId $templatePath ? In that case, do I have to add these:
```
<script id="event-short-list" type="text/html">
    {{ readFile ("/js/templates/event-list-format.mustache") | safeHTML}}
</script>
```

Reply:
It exists, but that doesn't mean it's added to every page. Those js/template files are optional and need to be asked to be included, and that's an intentional decision. We shouldn't be including content we don't need. The event list template was added as it was a common case for our themes. If you set the template id to `event-short-list` it should be able to target that shortcode in particular.

Reply:
Thanks! this works for me! I just made a push dropping unnecessary stuff.

```
    {{< newsroom/news
          title="News"
          titleClass="heading-line text-center"
          id="news-list-container"
          publishTarget="openmdm"
          count="5"
          templateId="news-no-mh"
          templatePath="node_modules/eclipsefdn-hugo-solstice-theme/js/templates/news-no-mh.mustache"
          includeList="true" >}}
```
```
    {{< newsroom/events
          title="events"
          titleClass="heading-line text-center"
          id="events-list-container"
          publishTarget="openmdm"
          count="5"
          templateId="event-list-format"
          templatePath="node_modules/eclipsefdn-hugo-solstice-theme/js/templates/event-list-format.mustache"
          includeList="true" >}}
```